crates/bevy_core_pipeline/src/lib.rs

```diff
@@ -337,23 +338,20 @@ pub fn extract_clear_color(clear_color: Res<ClearColor>, mut render_world: ResMu
 
 pub fn extract_core_pipeline_camera_phases(
     mut commands: Commands,
-    active_cameras: Res<ActiveCameras>,
+    query_camera_2d: Query<Entity, With<Camera2d>>,
+    query_camera_3d: Query<Entity, With<Camera3d>>,
 ) {
-    if let Some(camera_2d) = active_cameras.get(CameraPlugin::CAMERA_2D) {
-        if let Some(entity) = camera_2d.entity {
-            commands
-                .get_or_spawn(entity)
-                .insert(RenderPhase::<Transparent2d>::default());
-        }
+    for entity in query_camera_2d.iter() {
```

Comment:
Ooh intresting: will this enable multiple cameras more easily?

Reply:
Yes, the multiple_windows example needs no additional setup for tue second camera anymore.